testing/benchmark/main.tf

```diff
@@ -38,9 +37,6 @@ provider "ec" {}
 
```

Comment:
```terraform
module "tags" {
  source  = "../infra/terraform/modules/tags"
  project = "benchmarks"
}
```

I'm not sure about this. We're setting `project = benchmarks`, using the convention for `a team/shared owned resource` but benchmarks are actually always individually owned. Am I missing something ?

Reply:
That makes sense when the benchmark is launched locally, but if it's launched as a CI run, then that doesn't hold anymore since some resources may be left dangling, and we own it as a team since the CI scheduled run can't own that.

Reply:
I've pushed some changes to account for CI! Let me know what you think!